service.json

```diff
@@ -2,34 +2,15 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "github>reside-eng/renovate-config:group-dep-types",
-    "github>reside-eng/renovate-config",
-    "packages:react",
-    "group:graphql-toolsMonorepo",
-    "group:graphqlcodegeneratorMonorepo",
-    "group:storybookMonorepo",
-    "group:nextjsMonorepo",
-    "group:emotionMonorepo",
-    "group:apollo-serverMonorepo",
-    "group:apolloclientMonorepo",
-    "group:react-apolloMonorepo",
-    "group:react-dndMonorepo"
+    "github>reside-eng/renovate-config"
   ],
-  "platformAutomerge": true,
-  "rebaseWhen": "never",
   "packageRules": [
     {
       "description": "Group Fastify and Fastify scoped packages",
       "matchPackageNames": ["fastify", "fastify-plugin"],
       "matchPackagePrefixes": ["@fastify/"],
       "groupName": "Fastify packages",
       "groupSlug": "fastify-packages"
-    },
-    {
-      "description": "Group @mui/x-* packages",
-      "matchPackagePatterns": ["^@mui/x-"],
```

Comment:
This wasn't part of one of the built in presets before - is it now? or are we intentionally removing?

Reply:
I'm intentionally removing this because this config is for our services, not our UIs.

Reply:
Ahh gotcha - good yeah since @mui-x is not part of groupmonorepos